.editorconfig

```diff
@@ -0,0 +1,19 @@
+root = true
+
+[*]
+end_of_line = lf
+indent_style = space
+insert_final_newline = true
+trim_trailing_whitespace = true
+
+[*.rs]
+indent_size = 4
```

Comment:
Wouldn't that be handled by `rustfmt`? Just wondering. 😅

Reply:
I guessed it wouldn't hurt :shrug:

Reply:
True. It's unlikely that rustfmt will change its default indentation I guess, so nevermind. 😆